### PR TITLE
tf-m: Silence harmless version warning

### PR DESCRIPTION
**Pull request by SebastianBoe**
Silence harmless version warning for users that have tagged TF-M.

TF-M creates a very noisy warning when you tag it:

CMake Warning at cmake/version.cmake:28 (message):
  Actual TF-M version is not available from Git repository.  Settled to
  v2.0.0
Call Stack (most recent call first):
  CMakeLists.txt:22 (include)

### CODE REVIEW

File: modules/trusted-firmware-m/CMakeLists.txt [hunk 1]
```diff
@@ -264,6 +264,7 @@ if (CONFIG_BUILD_WITH_TFM)
       -DCMAKE_BUILD_TYPE=${TFM_CMAKE_BUILD_TYPE}
       -DTFM_PLATFORM=${CONFIG_TFM_BOARD}
       -DCONFIG_TFM_BUILD_LOG_QUIET=ON
+	  -DSILENCE_TFM_VERSION_WARNING=ON
```
[the remark this answers is not in the document]
Comment (SebastianBoe) [reply modules/trusted-firmware-m/CMakeLists.txt]: fixed

**Review by nashif — APPROVED** [review]

**Comment by SebastianBoe** [comment]
any chance of getting this merged?

**Comment by nashif** [comment]
need more reviewers: @d3zd3z  can you please?

**Comment by SebastianBoe** [comment]
@d3zd3z ?

**Comment by SebastianBoe** [comment]
anyone home?

**Review by de-nordic — APPROVED** [review]

**Pull request merged**